docs/hw/print_compute.md

```diff
@@ -3,7 +3,7 @@
 !!! attention "Notice"
     This is an autogenerated page containing links to STLs suitable for mounting various compute boards to the iRobot® Create® 3 regular mounting pattern.
 {% for key0, val0 in data|dictsort %}
-## {{ key0|replace("NVIDIA", "NVIDIA®")|replace("Jetson", "Jetson™")|replace("Pi", "Pi®") }}
+## {{ key0|replace("NVIDIA", "NVIDIA®")|replace("Jetson", "Jetson™")|replace("Pi", "Pi®")|replace("NXP", "NXP®") }}
```

Comment:
Will have to keep this line in mind for the next dev board...

Reply:
Well, luckily that just is for adding trademarks and keeping them out of the folder file structure.

Reply:
I suppose this could have been pulled out to a sub-function that ingests a "lookup file" of all possible trademarks to keep it all in one place?